uint8 flag_detect_blueprint		# 识别建筑图案完成标志位
float64[] blueprint			    # 建筑图矩阵
uint8 flag_update_blueprint		# 监督更新标志位
float64[] blueprint_update	    # 实时建筑图矩阵
uint8 flag_finished				# 任务结束标志位
float64 x						# 当前本机位置x坐标
float64 y						# 当前本机位置y坐标
float64 z						# 当前本机位置z坐标
float64 vx						# 当前本机速度x方向的大小
float64 vy						# 当前本机速度y方向的大小
float64 vrz						# 当前本机速度绕z方向的大小